src/trip/components/tripsModal.jsx

```diff
@@ -81,24 +104,26 @@ const TripsModal = ({ mode, initialData, disabled, floated, primary, isBack }) =
                         </Form.Field>
                     </Form.Group>
                     <Form.Field required>
-                        <label>Passagers</label>
+                        <label>Passagers</label>{
+                        userDropdownList ?
                         <Dropdown
                             placeholder='Selection des passagers'
                             fluid
                             multiple
                             search
                             selection
                             clearable
-                            options={DATA}
+                            options={userDropdownList}
```

Comment:
So maybe keeping user objects and using the correct property at the right time is smater than separating ids/usernames to then have to find the other back frequently

Reply:
The problem is that  "options" props expects a list of objects with following keys:  { Text, Value }, actually you are right, i keep the id as a value and the username as text for the dropdown.

What i could do is try use the map function directly on the "users" state object to create the { text, value } object, on the fly. I could also apply a filter to get rid of the driver to prevent from being able to select yourself as a passenger.

Therefore we wont need the userDropdownList state anymore, what do you think ?

Reply:
Yep, sounds good

Reply:
done